assessment/asr_validation/wald2017b/analysis/A1-biaxial.i

```diff
@@ -16,13 +18,9 @@
 
 [Variables]
   [T]
-    order = FIRST
-    family = LAGRANGE
     initial_condition = 10.6
```

Comment:
Do we want to change the initial condition for the temperature to 30 as well? You have modified the `stress_free_temperature` to 30.